content/action/how-can-i-contribute.md

```diff
@@ -124,38 +124,41 @@ THINGS YOU CAN OUTSIDE YOUR HOUSE
 THINGS YOU CAN DO AT WORK
 ---
 -   If you have a fruit or snack that you don’t want, don’t throw it out. Give it away to someone who needs and is asking for help.
-    
+
 -   Does everyone at work have access to healthcare? Find out what your rights are to work. Fight against inequality.
-    
+
 -   Mentor young people. It’s a thoughtful, inspiring and a powerful way to guide someone towards a better future.
-    
+
 -   Women earn 10 to 30 per cent less than men for the same work. Pay inequality persists everywhere. Voice your support for equal pay for equal work.
-    
+
 -   4 billion people lack access to basic sanitation services. Lend your voice to talk about the lack of toilets in many communities around the world!
-    
+
 -   Make sure your company uses energy efficient heating and cooling technology, and adjust the thermostat, lower in winter, higher in summer.
-    
+
 -   Stay informed. Read about workers in other countries and business practices. Talk to your colleagues about these issues.
-    
+
 -   Does your company invest in clean and resilient infrastructure? It’s the only way to keep workers safe and protect the environment.
-    
+
 -   Raise your voice against any type of discrimination in your office. Everyone is equal regardless of their gender, race, sexual orientation, social background and physical abilities.
-    
+
 -   Bike, walk or take public transport to work. Save the car trips for when you’ve got a big group.
-    
+
 -   Organize a No Impact Week at work. Learn to live more sustainably for at least a week: [un.org/sustainabledevelopment/be-the-change](http://un.org/sustainabledevelopment/be-the-change).
-    
+
 -   Speak up! Ask your company and Government to engage in initiatives that will not harm people or the planet. Voice your support for Paris Agreement!
-    
+
 -   Much of the waste that we produce on land ends up in the oceans.
-    
+
 -   Examine and change everyday decisions. Can you recycle at your workplace? Is your company buying from merchants engaging in harmful ecological practices?
-    
+
 -   Know your rights at work. In order to access justice knowing what you are entitled to will go a long way.
-    
+
 -   Corporate social responsibility counts! Encourage your company to work with civil society and find ways to help local communities achieve the goals.
 
 {{< padding >}}
 {{< padding >}}
 
-{{< button link="http://www.un.org/sustainabledevelopment/takeaction/" text="Lazy Person’s Guide to Saving the World" >}}
\ No newline at end of file
+{{< button link="http://www.un.org/sustainabledevelopment/takeaction/" text="Lazy Person’s Guide to Saving the World" >}}
+{{< padding >}}
+{{< facebook facebook-id="https://www.facebook.com/worldeconomicforum/videos/10155155806976479/" >}}
+*Video credit: World Economic Forum*
```

Comment:
You need to put a new line here, e.g.

{{< facebook facebook-id="https://www.facebook.com/worldeconomicforum/videos/10155155806976479/" >}}

*Video credit: World Economic Forum*